src/views/MediaChannels/PageView.tsx

```diff
@@ -33,7 +34,7 @@ const useStyles = makeStyles<Theme>(theme => ({
 interface PageProps {
   loading: boolean;
   hasMore: boolean;
-  criteria: { name?: string };
+  criteria: SearchProductCriteria;
   productList: Array<any>;
```

Comment:
Thanks!
Same for a product, don't you think? 🤔

Reply:
Not really seeing the benefit for that atm. It is not checked at runtime. Product is getting straight from the pkb.